### PR TITLE
AssetsApi with MultiLocation as preparation for multi pallet_assets instances

### PR DESCRIPTION
**Pull request by bkontur**

### CODE REVIEW

**Review by jsidorenko — APPROVED** [review]

**Comment by jsidorenko** [comment]
Niiice 👍

File: parachains/runtimes/assets/common/src/assets_api.rs [hunk 1]
```diff
@@ -18,9 +18,17 @@
 
 //! Runtime API definition for assets.
 
-use codec::Codec;
+use codec::{Codec, Decode, Encode};
+use frame_support::RuntimeDebug;
 use sp_std::vec::Vec;
 
+/// The possible errors that can happen querying the storage of a assets.
```
Comment (joepetrowski) [inline parachains/runtimes/assets/common/src/assets_api.rs:25]:
```suggestion
/// The possible errors that can happen querying the storage of assets.
```

**Review by joepetrowski — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2394760

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2394917

**Pull request merged**